service/league_service/league_rater.py

```diff
@@ -18,4 +20,100 @@ def rate(
         outcome: GameOutcome,
         player_rating: Rating,
     ):
-        return current_score
+        if current_score.game_count < config.PLACEMENT_GAMES - 1:  # This check is before we increase game_count
+            return LeagueScore(
+                current_score.division_id,
+                current_score.score,
+                current_score.game_count + 1
+            )
+
+        rating = player_rating[0] - 3 * player_rating[1]
```

Comment:
Askaholic is calling this `displayed_rating`, if you want you can adopt that terminology.

Reply:
this rating never actually gets displayed in the context of the league system. I get where you are coming from, but I don't think it is really a better name in the league rater.